esmvalcore/config-developer.yml

```diff
@@ -205,7 +210,9 @@ obs4mips:
   input_dir:
     default: 'Tier{tier}/{dataset}'
     RCAST: '/'
-  input_file: '{short_name}_{dataset}_{level}_{version}_*.nc'
+    IPSL: '{realm}/{short_name}/{freq}/{grid}/{institute}/{dataset}/{latest_version}'
+  input_file:
+    default: '{short_name}_{dataset}_{level}_{version}_*.nc'
```

Comment:
This seems like a change that is rather unrelated to this PR. Was the old version broken? Or is it just for homogeneity? Either way, I would probably rather see this change in a separate PR.

Reply:
You're right. It is not useful